web/src/enterprise/campaigns/detail/changesets/ChangesetNode.tsx

```diff
@@ -79,7 +80,10 @@ export const ChangesetNode: React.FunctionComponent<ChangesetNodeProps> = ({
                                 target="_blank"
                                 rel="noopener noreferrer"
                             >
-                                {node.title} (#{node.externalID})
+                                {node.title} (#{node.externalID}){' '}
+                                {node.externalURL && node.state !== ChangesetState.DELETED && (
+                                    <ExternalLinkIcon size="1rem" />
```

Comment:
Can we avoid the explicit size?
```suggestion
                                    <ExternalLinkIcon className="icon-inline" />
```

Reply:
icon-inline was way too big and dangling around weirdly 🤔

Reply:
It should take a size relative to the font size, so it would be expected (and desired) that it is bigger in a heading element.

What's the parent's `display`?

Reply:
It does do that, but it doesn't look good anyways. Given that it works for now, I'm going to put this into the 3.15 release and will see if I can find some other solution as a follow-up